setup.py

```diff
@@ -216,7 +216,7 @@ def get_extension_modules():
       packages = ['turbodbc'],
       extras_require={
           # We pin Apache Arrow quite restrictively until they guarantee a stable API
-          'arrow': ['pyarrow>=0.11,<0.12'],
+          'arrow': ['pyarrow>=0.11,<0.13'],
```

Comment:
@xhochy Would this still be compatible with pyarrow 0.11? If not, the requirements need to be changed